packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -2076,19 +2078,23 @@ export default class Meeting extends StatelessWebexPlugin {
               );
             };
 
-            // if a remote participant is stealing the presentation from us
-            if (
-              !this.mediaProperties.mediaDirection?.sendShare ||
-              oldShareStatus === SHARE_STATUS.WHITEBOARD_SHARE_ACTIVE
-            ) {
+            try {
+              // if a remote participant is stealing the presentation from us
+              if (
+                this.mediaProperties.mediaDirection?.sendShare &&
+                oldShareStatus === SHARE_STATUS.LOCAL_SHARE_ACTIVE
+              ) {
+                if (this.isMultistream) {
```

Comment:
at what step will we remove the updateShare public method and have the unpublish trigger a event then updateShare gets called internally

Reply:
we will remove updateShare() and all other methods listed on my wiki only after we have publish/unpublish API working with webrtc-core classes for both transcoded and multistream.